src/main/java/com/github/yu55/gog/handler/git/GitAnnotateHandler.java

```diff
@@ -33,7 +35,10 @@ public AnnotateResponse annotate(AnnotateRequest annotateRequest) {
 
     private AnnotateResponse executeAndConvertToResponse(GitRepository gitRepository,
                                                          GitCommandExecutor gitCommandExecutor) {
-        List<String> annotatedOutputLines = gitCommandExecutor.execute(gitRepository.getDirectory());
+        List<String> annotatedOutputLines = gitCommandExecutor.execute(gitRepository.getDirectory()).getOutputLines()
```

Comment:
this change is a step in refactoring iteration that allows me not to break working functionality. It's clear that the next step would implement this as you mentioned;)